recipes/sracat/build.sh

```diff
@@ -0,0 +1,7 @@
+make \
+    SRA_PATH=${PREFIX} \
```

Comment:
Should this be
```suggestion
    SRA_PATH=${BUILD_PREFIX} \
```
instead?